docs/central-users.rst

```diff
@@ -111,16 +151,16 @@ Newly created Web Users are only able to log in and edit their profile informati
 
 .. _central-users-web-role:
 
-Assigning a site-wide Web User Role
+Assigning Web User Roles
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-As mentioned under :ref:`central-users-web-roles` above, there are four Roles you may assign to Web Users in the current release of ODK Central: Administrator, Project Manager, Project Viewer, and Data Collector. Administrators may perform any action on the system, while Project Managers may perform any action on their assigned Project(s). Project Viewers may only see created forms and submissions within the Project they are assigned to, and cannot edit anything. Data Collectors may see the list of all Forms in the Project, and retrieve basic information about them, but may only create new Submissions. They cannot see any Submissions.
+As mentioned under :ref:`central-users-web-roles` above, there are four Roles you may assign to Web Users in the current release of ODK Central: Administrator, Project Manager, Project Viewer, and Data Collector. 
 
-To learn how to assign a Project Manager or Viewer role, please see the :ref:`central-project-roles` section in the Projects guide.
+To learn how to assign Project-level roles, please see the :ref:`central-project-roles` section in the Projects guide.
```

Comment:
```suggestion
To learn how to assign Project-level Roles, please see the :ref:`central-project-roles` section in the Projects guide.
```

Reply:
Suggesting this for consistency with other places where we capitalize "role". That said, I don't feel certain that we need to capitalize "role" anywhere. It's not a concept that has a special meaning in Central, so I think it'd be OK not to.

Reply:
Oh wait, I guess we do capitalize "role" in Frontend. I don't think we need to change Frontend. So yeah, I guess we should capitalize here as well.